app/services/activities/import_from_csv.rb

```diff
@@ -259,6 +261,12 @@ def convert_parent_id(roda_id)
         parent.id
       end
 
+      def convert_covid19_related(covid19_related)
+        raise I18n.t("importer.errors.activity.invalid_covid19_related") unless covid19_related_radio_options.map(&:code).include?(covid19_related.to_i)
```

Comment:
Ruby helpfully converts (some?) non-digit strings into 0, thus making the input appear valid. For example:

```
 [0, 1, 2, 3, 4].include?("bla".to_i)
=> true
```

Are we OK with that? Would it be ok to convert the options to strings and then check that the input is included in the stringified options?

Reply:
Good shout. I've reversed this, so we cast both sides to strings, so it's more reliable.